service_mesh/service_mesh_day_two/ossm-auto-route.adoc

```diff
@@ -0,0 +1,66 @@
+[id="ossm-auto-route"]
+= Automatic route creation
+include::modules/ossm-document-attributes.adoc[]
+toc::[]
+
+OpenShift routes for Istio Gateways are automatically managed in {ProductName}. Every time an Istio Gateway is created, updated or deleted inside the service mesh, an OpenShift route is created, updated or deleted. 
+
+[id="ossm-auto-route-enable_{context}"]
+== Enabling Automatic Route Creation
+A {ProductName} control plane component called Istio Openshift Routing (IOR) synchronizes the gateway route. You can deploy it as part of the control plane in the {ProductName} installation process. For more information, see xref:../../service_mesh/service_mesh_install/customizing-installation-ossm.adoc#ossm-cr-gateway_customizing-installation-ossm[Istio gateway configuration].
```

Comment:
is it worth highlighting that IOR is disabled by default?

Reply:
Yes, I added a phrase that says you must enable it.